docs/de/404.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Please delete this & find a solution that uses the same 404 page for every language. I'd prefer not to add the burden of translating the 404 page into every language.

Even in you're current solution, this page is never reached. E.g. when I navigate to https://deploy-preview-310--ethereumorg.netlify.com/de/asdfasdf it redirects to the English 404.

Reply:
Seeing that the rest of the site is translated, why not translate the 404 page? It's not mandatory though, if no specific `404.md` is placed in a language subdir, then the english one from the root dir will be used.

It worked reliably on my local dev env. I'll check again and see what's going wrong here about the 404 page not being reached.

Reply:
Mmm ok, I can support that! Use the language-specific 404, otherwise fallback to English if it doesn't exist.

Again, I think it's also important to retain the URL that the user typed in (i.e. maintain, that current UX):
> Can you also find a solution that retains the URL the user attempted to reach (vs. redirecting them to 404.html)? That's typically a best practice so the user can see the URL that was broken.

Reply:
In other words, if the `NotFound.vue` maintains the URL, I'd rather go that route. We can sort out the language-specific copy in a similar fashion to the HomePage component, by using translations.js.

Reply:
I'll check if I can maintain the URL with `404.md`.

IIRC with the `NotFound.vue` approach there was no easy support for the sites global theme. 

Will check all the points you mentioned in the next few days and update you.

Reply:
@samajammin I got it working with `NotFound.vue`, keeping the original URL. Put the script from `Layout.vue` into a separate file, because it's also needed in `NotFound.vue` when using the sites global theme. I hope that's ok.

For some reason now the "Not Found" page is always in english, including the menu. Since you didn't want it translatable earlier, should I put more effort into fixing this?